ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py

```diff
@@ -831,18 +831,10 @@ def test_jax_numpy_std(
 def test_jax_numpy_corrcoef(
     dtype_and_x,
     rowvar,
-    frontend,
-    test_flags,
-    fn_tree,
-    on_device,
 ):
     input_dtypes, x = dtype_and_x
     np_helpers.test_frontend_function(
         input_dtypes=input_dtypes,
-        frontend=frontend,
```

Comment:
Same here?

Reply:
the last time i resolved the conflict using the web and this apparently caused some line to be removed
today i used pycharm ide and on my fork every thing is perfect